kaitai_utils.py

```diff
@@ -2,13 +2,28 @@
 import importlib
 import inspect
 from io import BufferedWriter
-import json
-from json_stream import streamable_dict, streamable_list
 import os
 from typing import Any, BinaryIO, Dict, Generator, List, Type
-import yaml
 
+import hansken_extraction_plugin
+from hansken_extraction_plugin.api.extraction_trace import ExtractionTrace
+import json
+from json_stream import streamable_dict, streamable_list
 from kaitaistruct import KaitaiStruct
+import yaml
+
+"""
+This method creates traces from a kaitai object tree. It results in a trace containing a JSON representation of the
```

Comment:
Which method does this refer to? I think pydoc is usually inside the method, right?

Reply:
Fixed!